### DISCUSSION TITLE
Add typing support

### DISCUSSION THREAD

**Discussion by BowlBird** [spacedriveapp/spacedrive#1519] [Feature Request]
Hello!

I'm really excited for this project, and I think this would make it much quicker to use. So, for most file browsers on Linux (although I'm sure others support it), while focusing the window, you have the ability to just start typing and the file explorer will highlight files for you that match what you have typed.

I know there is a search feature, but this would more match immediate files in a directory with no fuzzy search.

Additionally, if you type '/' or '~', most file explorers take you to the root and home directory and allow you type a full path after those key characters. I think this would be a great feature to add as well, as these are very common directories. I don't know how other systems would interact with this (such as android, windows, or ios), but maybe they could do something similar. I also am unsure of how this would work with having multiple root directories (multiple machines with space drive), but maybe it could just be localized to that machine once selected or something.

Thanks!

**Comment by utkubakir** [comment]
Hey,

Thank you for the feedback, we already had this on our todo-list but I will make sure to add your comments on it so we know what to aim for!

Thanks.
